src/comments.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/named */
+/* eslint-disable import/extensions */
+
+import { pokeCall } from './apiCall.js';
+import close from './cancel.png';
```

Comment:
I kindly suggest comparing the conformity of the imported file source with the name of the file because it can not be loaded.

Reply:
I removed that line and included the cancel.png in the proper place.